src/util.js

```diff
@@ -38,22 +40,37 @@ export async function ensureDir(dir) {
     }
 }
 
+/**
+ * @param {string} value
+ */
+const parseTxtRecordData = (value) => {
+    const result = {};
+    for (const part of value.split(';')) {
+        const [key, value] = part.split('=');
+        if (key && value) {
+            result[key] = value;
+        }
```

Comment:
```suggestion
        const [key, ...value] = part.split('=');
        if (key && value.length > 0) {
            result[key] = value.join('=');
        }
```
This will leave something like `forward-domain=https://a.com?b=c` remain works.

Reply:
Good catch!

Reply:
Done.